src/main/build/debug.c

```diff
@@ -124,4 +124,5 @@ const char * const debugModeNames[DEBUG_COUNT] = {
     [DEBUG_GIMBAL] = "GIMBAL",
     [DEBUG_WING_SETPOINT] = "WING_SETPOINT",
     [DEBUG_AUTOPILOT_POSITION] = "AUTOPILOT_POSITION",
+	[DEBUG_AOA_ESTIMATOR] = "AOA_ESTIMATOR",
```

Comment:
@demvlad please check your TAB settings. Should be 4 spaces. There are other lines affected.

Reply:
@demvlad `DEBUG_AOA_ESTIMATOR` also needs to be added to debug.h